h/models/token.py

```diff
@@ -48,10 +58,13 @@ class Token(Base, mixins.Timestamps):
     #: A NULL value means it is a developer token.
     authclient = sqlalchemy.orm.relationship('AuthClient')
 
-    def __init__(self, **kwargs):
-        super(Token, self).__init__(**kwargs)
+    def __init__(self, expires=None, **kwargs):
```

Comment:
I can certainly pull `expires` out of `kwargs` instead as you suggest. Does this make any difference to the method signature though? (Both ways, `expires` is an optional arg.)

Also wondering what you'll think of https://github.com/hypothesis/h/pull/4323/commits/7a2d697ae769675c5aa335c25745e9f4a4e764a6 (which is in [another pr](https://github.com/hypothesis/h/pull/4323/commits))